Source/LightboxController.swift

```diff
@@ -212,9 +212,9 @@ open class LightboxController: UIViewController {
   override open func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
     super.viewWillTransition(to: size, with: coordinator)
 
-    coordinator.animate(alongsideTransition: { (UIViewControllerTransitionCoordinatorContext) -> Void in
+    coordinator.animate(alongsideTransition: { (_) -> Void in
```

Comment:
I think you can replace `(_) -> Void in` with `_ in`.

Reply:
Done!